lib/tests/unit/modules/script_test.cc

```diff
@@ -113,29 +115,25 @@ TEST_CASE("Modules::Script::hasAction", "[modules]") {
 TEST_CASE("Modules::Script can execute a script", "[modules]") {
     Modules::Script mod { PXP_AGENT_BIN_PATH, MASTER_URIS, CA, CRT, KEY, "", 10, 20, MODULE_CACHE_DIR, STORAGE };
     SECTION("script executes and writes to stdout") {
-        auto response = mod.executeAction(script_request(TESTING_SCRIPT, TESTING_SCRIPT_SHA265, ""));
+        auto args = std::vector<std::string>();
+        auto response = mod.executeAction(script_request(TESTING_SCRIPT, TESTING_SCRIPT_SHA265, args));
         boost::trim(response.output.std_out);
         REQUIRE(response.output.std_out == "TESTING");
         REQUIRE(response.output.std_err == "");
         REQUIRE(response.output.exitcode == 0);
     }
     SECTION("script executes with multiple parameters") {
-        auto response = mod.executeAction(script_request(TESTING_SCRIPT, TESTING_SCRIPT_SHA265, "THREE TEST ARGS"));
+        auto args = std::vector<std::string>({"THREE", "TEST", "ARGS"});
+        auto response = mod.executeAction(script_request(TESTING_SCRIPT, TESTING_SCRIPT_SHA265, args));
         boost::trim(response.output.std_out);
         REQUIRE(response.output.std_out == "| THREE | TEST | ARGS");
         REQUIRE(response.output.std_err == "");
         REQUIRE(response.output.exitcode == 0);
     }
-    SECTION("script executes with params with quotes and spaces") {
```

Comment:
Array of strings lets you easily do this, so I don't think this test is necessary anymore.